Dockerfile

```diff
@@ -3,13 +3,8 @@ FROM rust:1.54.0-alpine as builder
 # Check cargo version
 RUN cargo --version
 
-# Download binaryen and verify checksum
-ADD https://github.com/WebAssembly/binaryen/releases/download/version_96/binaryen-version_96-x86_64-linux.tar.gz /tmp/binaryen.tar.gz
-RUN sha256sum /tmp/binaryen.tar.gz | grep 9f8397a12931df577b244a27c293d7c976bc7e980a12457839f46f8202935aac
-
-# Extract and install wasm-opt
-RUN tar -xf /tmp/binaryen.tar.gz
-RUN mv binaryen-version_*/wasm-opt /usr/local/bin
+# Install platform-specific binaryen
+RUN apk update && apk add binaryen
```

Comment:
Nice.

I'm a bit concerned we don't control the version anymore but great to have the multi-arch issue covered this way.

Reply:
We can always do `apk add binaryen=98-r0`.